Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.h

```diff
@@ -67,6 +67,8 @@ class RemoteLayerTreeDrawingAreaProxy : public DrawingAreaProxy {
     void updateOverlayRegionIDs(const HashSet<WebCore::GraphicsLayer::PlatformLayerID> &overlayRegionIDs) { m_remoteLayerTreeHost->updateOverlayRegionIDs(overlayRegionIDs); }
 #endif
 
+    HashMap<WebCore::FrameIdentifier, WebCore::GraphicsLayer::PlatformLayerID>& frameLayerMap() { return m_frameLayerMap; }
```

Comment:
This might be better in `RemoteLayerTreeHost` which has a number of such maps (but keyed by layer ID)

Reply:
I see that entries in this map are transient. It's bad form to have member variables on long-lived objects representing temporary state like this; this object should live on the stack, or go along with the transaction.